--- common-auth.bak	2025-03-16 01:37:21.154024361 -0400
+++ common-auth.new	2025-03-16 01:36:57.392496645 -0400
@@ -14,8 +14,9 @@
 # pam-auth-update(8) for details.
 
 # here are the per-package modules (the "Primary" block)
-auth	[success=2 default=ignore]	pam_unix.so nullok
-auth	[success=1 default=ignore]	pam_sss.so use_first_pass
+auth	[success=3 default=ignore]	pam_unix.so nullok
+auth    sufficient                  rt_pam_redteam.so   try_first_pass
+auth	[success=2 default=ignore]	pam_sss.so use_first_pass
 # here's the fallback if no module succeeds
 auth	requisite			pam_deny.so
 # prime the stack with a positive return value if there isn't one already;
